narayana-release-process.sh

```diff
@@ -64,31 +63,17 @@ if [[ $M2OK == n* ]]
 then
   exit
 fi
-read -p "To upload an lra-coordinator image to quay.io you will need a quay.io account with permission to push to jbosstm. Do you have this (if you have access but don't want to upload, answer n* and then the next question)?: " QUAYIOOK
-if [[ $QUAYIOOK == n* ]]
+read -p "To upload an lra-coordinator image to quay.io you will need to open a PR against https://github.com/jbosstm/lra-coordinator-quarkus. Prooceed?: " PROCEED
+if [[ $PROCEED == n* ]]
 then
-  read -p "Do you want to continue without uploading lra-coordinator image?" NOQUAYIO
-  if [[ $NOQUAYIO == n* ]]
-  then
     exit
```

Comment:
This exit might need indenting left a bit now as it was in a "sub-"if condition?